src/utils/exportUtils/Exporter.ts

```diff
@@ -77,7 +77,7 @@ export default abstract class Exporter {
 
     protected async downloadZIP(): Promise<string | void> {
         const brand = SdkConfig.get().brand;
-        const filenameWithoutExt = `${brand} - Chat Export - ${formatFullDateNoDay(new Date())}`;
+        const filenameWithoutExt = `${brand} - ${this.room.name} - Chat Export -${formatFullDateNoDayISO(new Date())}`;
```

Comment:
I think we still need to consider the room name having invalid characters in it?

applies to later on as well.

Reply:
@turt2live there is another PR which does that, I believe: #7992

Reply:
We can't land this code at the moment without that code, given the safety risk we'd be taking on. It feels like it might have been split up too much?